apps/consent-das/src/routers/consentQuestions.ts

```diff
@@ -159,7 +159,7 @@ router.post('/', async (req, res) => {
 	const { id, isActive, category } = req.body;
 	// TODO: add validation
 	try {
-		const question = await createConsentQuestion({ id, isActive, category });
+		const question = await createConsentQuestion({ consentQuestionId: id, isActive, category });
```

Comment:
And then this line would become:

```suggestion
		const question = await createConsentQuestion({ consentQuestionId, isActive, category });
```

Reply:
(I suppose the Postman collection might need updating after this too) 🙈